go.mod

```diff
@@ -14,7 +14,7 @@ require (
 	github.com/kuadrant/authorino-operator v0.11.1
 	github.com/kuadrant/dns-operator v0.0.0-20241002074817-d0cab9eecbdb
 	github.com/kuadrant/limitador-operator v0.9.0
-	github.com/kuadrant/policy-machinery v0.2.0
+	github.com/kuadrant/policy-machinery v0.5.0
```

Comment:
I hadn't realised `main` was so much behind. In fact, this explains why I was observing multiple stale update events related to the topology ConfigMap at [another branch](https://github.com/Kuadrant/kuadrant-operator/tree/sotw/rlp) but not really in `main`.

Added another commit, shamelessly [stolen](https://github.com/Kuadrant/kuadrant-operator/pull/885) from @KevFan, to widen the change to all kinds of resources watched by the SOTW reconciler.